java-to-zod-core/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>sh.ivan</groupId>
     <artifactId>java-to-zod</artifactId>
-    <version>0.7.0-SNAPSHOT</version>
+    <version>0.8.0-SNAPSHOT</version>
```

Comment:
No need to change the version number. This is all handled automatically when I do the release